libraries/src/Toolbar/ToolbarHelper.php

```diff
@@ -43,7 +43,12 @@ public static function title($title, $icon = 'generic.png')
 
 		$app = Factory::getApplication();
 		$app->JComponentTitle = $html;
-		Factory::getDocument()->setTitle(strip_tags($title) . ' - ' . $app->get('sitename') . ' - ' . Text::_('JADMINISTRATION'));
+		$title = strip_tags($title) . ' - ' . $app->get('sitename');
+		if ($app->isClient('administrator'))
```

Comment:
```suggestion

		if ($app->isClient('administrator'))
```
See [https://ci.joomla.org/joomla/joomla-cms/33784/1/9](https://ci.joomla.org/joomla/joomla-cms/33784/1/9)